### ISSUE TITLE
SyntaxError: f-string expression part cannot include a backslash

### ISSUE THREAD

**Issue by G-loy** [a-s-g93/neo4j-runway#74]
neo4j_runway/ingestion/cypher.py:255
It seams `res += f"  - {cql.lstrip().replace('\n', '\n    ')}\n"`  caused an error
I move the part containing the back slash outside the string temporarily solves this problem
```
cql_formatted = cql.lstrip().replace('\n', '\n    ')
res += f"  - {cql_formatted}\n"
```

**Comment by a-s-g93** [comment]
Hi G-loy, Can you please provide some more information regarding the environment you're running in? Thank you.

**Comment by a-s-g93** [comment]
This code has been implemented and tested with Python 3.10+. Fix is present in runway v0.5.1

**Comment by G-loy** [comment]
> Hi G-loy, Can you please provide some more information regarding the environment you're running in? Thank you.
anconda  environment
Python 3.10.14
